scripts/cloud.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+source .envrc
+
+# Generate report of current env vars
+echo "======================================================"
+printenv | grep "KKA_.*"
+echo "======================================================"
+
+if [ "${KKA_DEPLOY_MINIMAL}" == "false" ]; then
+  # Install ArgoCD
+  kubectl create namespace argocd
+  helm repo add argo https://argoproj.github.io/argo-helm
+  helm repo update
+  helm install argocd argo/argo-cd --namespace=argocd -f ./addons/argocd/values.yaml
+  sleep 10
+  # Install apps
+  kubectl apply -f https://github.com/nearform/k8s-kurated-addons/releases/download/v0.0.1/app-of-apps.yaml
```

Comment:
I think this can't be hardcoded this way, it makes local development hard to work from a branch but also not sure if the release please can handle it.

Reply:
I'm not sure too. I will learn more about our release phase to understand it.

Reply:
TBF, I don't think we need this file at all.

Reply:
Pointed Makefile to install only argocd to the pointed cluster.